src/core/render/index.js

```diff
@@ -294,7 +294,8 @@ export function renderMixin(proto) {
   };
 
   proto._renderNav = function(text) {
-    text && this._renderTo('nav', this.compiler.compile(text));
+    text &&
+      this._renderTo(this.config.nav_el || 'nav', this.compiler.compile(text));
```

Comment:
- [ ] We should follow our `camelCase` convention for non-class identifiers: `nav_el` -> `navEl`.